ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Test.hs

```diff
@@ -184,8 +184,8 @@ instance (Eq header, Eq point) => Eq (AnyMessage (ChainSync header point)) where
   AnyMessage (MsgRollForward h1 p1)       == AnyMessage (MsgRollForward h2 p2)       = h1 == h2 && p1 == p2
   AnyMessage (MsgRollBackward p1 h1)      == AnyMessage (MsgRollBackward p2 h2)      = p1 == p2 && h1 == h2
   AnyMessage (MsgFindIntersect ps1)       == AnyMessage (MsgFindIntersect ps2)       = ps1 == ps2
-  AnyMessage (MsgIntersectFound p1 h1)    == AnyMessage (MsgIntersectFound p2 h2) = p1 == p2 && h1 == h2
-  AnyMessage (MsgIntersectUnchanged h1)   == AnyMessage (MsgIntersectUnchanged h2)   = h1 == h2
+  AnyMessage (MsgIntersectFound p1 h1)    == AnyMessage (MsgIntersectFound p2 h2)    = p1 == p2 && h1 == h2
+  AnyMessage (MsgIntersectNotFound h1)    == AnyMessage (MsgIntersectNotFound h2)    = h1 == h2
```

Comment:
Could you remove a few spaces of indentation before the `=` sign.

Reply:
OK